src/components/Daos/DaoCard.tsx

```diff
@@ -34,18 +33,13 @@ const DaoCard = (props: IProps) => {
       <div className={classNames({
         [css.dao]: true,
         [css.daoInactive]: inActive})}>
-        <div className={css.daoTitle}
-          style={{backgroundImage: bgPattern.toDataUrl()}}>
-
+        <div className={css.daoTitle} style={{backgroundColor: bgPattern.color}}>
```

Comment:
We should not be setting styles inline, only in CSS

Reply:
The colors come from a third-party module in the JavaScript and if we want to use it we need to apply it from the JavaScript code. As you can see, also the old code worked in that way.